### PR TITLE
add ac101 codec (AUD-622)

### PR DESCRIPTION
**Pull request by donny681**
Hi,I'm an enginner of Aithinker.We will publish an ESP32 audo kit.The codec of the board is AC101.We plan to add our  codes of AC101 to ESP-ADF and be compatible with your program.I test our audio kit and ESP32 LYRAT Board.The program works well.We will help you to develop ESP-ADF.

### CODE REVIEW

**Comment by jason-mao** [comment]
@donny681 Thanks for your PR. Could you provide AC101 codec board to us? I have sent mail to you several days ago, but not received your reply.

**Comment by donny681** [comment]
OK,we will product new boards next week.We will send new board to you as soon as possible.

File: components/audio_hal/Kconfig.projbuild [hunk 1]
```diff
@@ -0,0 +1,19 @@
+menu "BOARD"
+choice BOARD
+    prompt "BOARD"
```
Comment (projectgus) [inline components/audio_hal/Kconfig.projbuild]: Please don't use all-capitals for this text and the "menu" text. Suggest "Audio Board" not "BOARD"

Comment (donny681) [reply components/audio_hal/Kconfig.projbuild]: OK,Thanks for your advice.

**Comment by fsicre** [comment]
Could you finish this merge/pull-request ?

**Comment by humberto-trujillo** [comment]
Hello, I recently got an Aithinker board and I have a couple of questions about this PR.
I've been trying the examples in the SDK but they don't seem to work even though I selected the Audio HAL to be AI-ESP32, is there any other configuration I need to do? Currently, it is complaining about not being able to read or write to the ES8388 codec, which I believe is what LyraT uses.
Are there any other examples in this PR that work on this board?

**Comment by aguynamedben** [comment]
+1 🙏

**Comment by aguynamedben** [comment]
@humberto-trujillo @fsicre Damn, I just got my Aithinker ESP32-A1S and didn't realize the Aithinker boards used a different audio chip that doesn't have a driver in esp-adf already.

Did y'all end up just buying a LyraT? Or using some form of this?

Is there any chance this is going to be merged soon/ever, or should I just give up now and buy a LyraT?

**Comment by tcpipchip** [comment]
any news about ?

**Comment by jason-mao** [comment]
Closed, please refer to https://github.com/espressif/esp-adf/issues/341

**Pull request closed without merging**